ci/pipelines/s3-resource-simple-template.yml

```diff
@@ -92,51 +78,25 @@ resources:
     region_name: eu-west-1
     access_key_id: ((concourse_user_access_key))
     secret_access_key: ((concourse_user_secret_key))
+    # For this particular pipeline we use the /artifacts paths also for the master branch,
+    # so that the files uploaded by the tests will expire and be removed periodically.
     path: artifacts/s3-resource-simple/((branch))/
-    sync: false # Disable downloading.
-    options:
-    # FIXME! `--recursive` is _currently_ needed to copy a directory :-(
-    - --recursive
+    # DANGER DANGER DANGER
+    # For vast the majority of your use cases, DO NOT enable downloading (do NOT set sync: true)!
+    # You will download, on each get (explicit and implicit) EVERYTHING contained in the bucket!
```

Comment:
- in the bucket or under the path? 
- just dowload or sync? which means it can also delete in the bucket

Reply:
- under the path. I will fix this, thanks.
- "sync" is a misnamed feature. It does not delete anything, it just downloads everything :-/

Reply:
Unless you pass `--delete` to it, so it's not that badly named. But it's nice that it works in a safe way by default.

Reply:
I think that passing --delete will always delete everything, since it is performed during a get step, so the destination will always start empty...

Reply:
Updated the explanation mentioning the "path".